.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no grunt configure && npx --no lint-staged
```

Comment:
Why put this on the same line?

Also, should we stage rule-descriptions.md? We can call `grunt configure` all we want, if we don't stage it, it won't end up in the commit.

Reply:
> if we don't stage it, it won't end up in the commit

Doesn't lint-staged take care of this?

Reply:
`lint-staged` only runs on files that are staged. If you run `npx lint-staged` without staging anything, it outputs `No staged files found.`

These commands existed on the same line in the original hook. Since `husky` will fail the commit if any command returns non-zero, I don't think there's any practical difference. I agree that it would be more clear to have them on different lines because it currently implies something that isn't really there using `&&`.